.github/workflows/nbval.yml

```diff
@@ -188,7 +188,7 @@ jobs:
       if: runner.os == 'MacOS'
       run: |
         chmod +x .ci/validate_notebooks.sh
-        .ci/validate_notebooks.sh 107-speech-recognition-quantization 114-quantization-simplified-mode 115-async-api
+        .ci/validate_notebooks.sh 107-speech-recognition-quantization 114-quantization-simplified-mode
```

Comment:
Unfortunately, it looks like this doesn't fix the bug for mac. So, we need to revert this change.

Reply:
I've reported a bug in OpenVINO